src/en/credentials.md

```diff
@@ -213,20 +207,48 @@ juju add-credential azure -f mycreds.yaml
     
 ## Managing credentials
 
-There are several management tasks that can be done related to credentials.
+Here we cover the management of credentials.
 
-### Listing credentials
+ - [Setting default credentials][#setting-default-credentials]
+ - [Listing local credentials][#listing-local-credentials]
+ - [Listing remote credentials][#listing-remote-credentials]
+ - [Updating local credentials][#updating-local-credentials]
+ - [Updating remote credentials][#updating-remote-credentials]
+ - [Removing local credentials][#removing-local-credentials]
+ - [Setting a remote credential for a model][#setting-a-remote-credential-for-a-model]
 
-When credentials are added to Juju they become available to use on a controller
-and its models. There are therefore two categories of credentials: those that
-are available and those that are currently in use.
+### Setting default credentials
+
+To set the default credential for a cloud:
 
-#### Available
+```bash
+juju set-default-credential aws carol
+```
 
-You can display what credentials are available by running the command:
+If only one credential exists for a cloud, it becomes the effective default
+credential for that cloud.
+
+Setting a default affects operations that require a credential such as creating
+a controller (`bootstrap`) and adding a model (`add-model`). It does not change
+what is currently in use (on a controller). With both these commands a
```

Comment:
Thank you! The lack of these being stated explicit was/is the cause of a lot of pain for users.